### PR TITLE
Add textbooks

### PR DESCRIPTION
**Pull request by jackdolgin**
Added a link to a list of math textbooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +136,9 @@ A curated list of awesome mathematics resources.
 * [Paul's Online Math Notes](http://tutorial.math.lamar.edu/) - Paul Dawkins (Lamar University)
 * [List of electronic textbooks](http://faculty.atu.edu/mfinan/nnotes.html) - Marcel B. Finan (Arkansas Tech University)
 * [Topology Atlas](http://at.yorku.ca/topology/)
-
+* [Recreations in Math](http://djm.cc/library/Recreations_in_Mathematics_Licks_edited.pdf) - H. E. Licks
```
Comment (rossant) [inline README.md:139]: Perhaps you could add the year for old scanned books like this one? e.g.

` * [Recreations in Math](http://djm.cc/library/Recreations_in_Mathematics_Licks_edited.pdf) - H. E. Licks (1917)`

File: README.md [hunk 2]
```diff
@@ -134,7 +136,9 @@ A curated list of awesome mathematics resources.
...
+* [Magic Squares and Cubes](http://djm.cc/library/Magic_Squares_Cubes_Andrews_edited.pdf) - W. S. Anderson
```
Comment (rossant) [inline README.md:140]: idem (1917)
Also the author is W. S. Andrews

File: README.md [hunk 3]
```diff
@@ -224,6 +229,10 @@ A curated list of awesome mathematics resources.
 
 * [A Course in Universal Algebra](http://www.math.uwaterloo.ca/~snburris/htdocs/ualg.html) - S. Burris, H.P. Sankappanavar
 * [A Course in Commutative Algebra](https://faculty.math.illinois.edu/~r-ash/ComAlg.html) - Robert Ash
+* [First Course in Algebra](http://djm.cc/library/First_Algebra_Hawkes_Luby_Touton_edited.pdf) - Herbert E. Hawkes, William A. Luby, Frank C. Touton (PDF)
```
Comment (rossant) [inline README.md:232]: idem (1909)

File: README.md [hunk 4]
```diff
@@ -224,6 +229,10 @@ A curated list of awesome mathematics resources.
...
+* [Second Course in Algebra](http://djm.cc/library/Second_Algebra_Hawkes_Luby_Touton_edited.pdf) - Herbert E. Hawkes, William A. Luby, Frank C. Touton
```
Comment (rossant) [inline README.md:233]: idem (1911)

File: README.md [hunk 5]
```diff
@@ -224,6 +229,10 @@ A curated list of awesome mathematics resources.
...
+* [Algebra: An Elementary Text-Book, Part I](http://djm.cc/library/Algebra_Elementary_Text-Book_Part_I_Chrystal_edited.pdf) - G. Chrystal (PDF)
```
Comment (rossant) [inline README.md:234]: idem (1904)

File: README.md [hunk 6]
```diff
@@ -224,6 +229,10 @@ A curated list of awesome mathematics resources.
...
+* [Algebra: An Elementary Text-Book, Part II](http://djm.cc/library/Algebra_Elementary_Text-Book_Part_II_Chrystal_edited02.pdf) - G. Chrystal (PDF)
```
Comment (rossant) [inline README.md:235]: idem (1900)

File: README.md [hunk 7]
```diff
@@ -298,6 +309,11 @@ A curated list of awesome mathematics resources.
 * [Elements](http://aleph0.clarku.edu/~djoyce/java/elements/toc.html) - Euclid
 * [Euclid's Elements Redux](http://starrhorse.com/euclid/) - Daniel Callahan
 * [Mathematical Illustrations](http://www.math.ubc.ca/~cass/graphics/manual/) - Bill Casselman
+* [Byrne's Euclid](https://www.c82.net/euclid/) - Oliver Byrne
+* [Plane Geometry](http://djm.cc/library/Plane_Geometry_Wentworth_Smith_edited.pdf) - George Wentworth and David Eugene Smith
```
Comment (rossant) [inline README.md:313]: idem (1913)

File: README.md [hunk 8]
```diff
@@ -298,6 +309,11 @@ A curated list of awesome mathematics resources.
...
+* [Planes and Spherical Trigonometry](http://djm.cc/library/Plane_Spherical_Trigonometry_Wentworth_Smith_edited_2.pdf) - George Wentworth and David Eugene Smith
```
Comment (rossant) [inline README.md:314]: idem (1915)

File: README.md [hunk 9]
```diff
@@ -298,6 +309,11 @@ A curated list of awesome mathematics resources.
...
+* [Coordinate Geometry](http://djm.cc/library/Coordinate_Geometry_Fine_Thompson_edited03.pdf) - Henry Buchard Fine and Henry Dallas Thompson (PDF)
```
Comment (rossant) [inline README.md:315]: idem (1911)

File: README.md [hunk 10]
```diff
@@ -298,6 +309,11 @@ A curated list of awesome mathematics resources.
...
+* [Analytic Geometry](http://djm.cc/library/Analytic_Geometry_Siceloff_Wentworth_Smith_edited.pdf) - Lewis Parker Siceloff, George Wentworth, David Eugene Smith
```
Comment (rossant) [inline README.md:316]: idem (1922)

File: README.md [hunk 11]
```diff
@@ -365,6 +381,8 @@ A curated list of awesome mathematics resources.
 * [Introduction to Calculus I and II](http://www.math.odu.edu/~jhh/counter10.html) - J.H. Heinbockel
 * [Active Calculus](http://faculty.gvsu.edu/boelkinm/Home/Active_Calculus.html) - Matt Boelkins
 * [Supplements to the Exercises in Chapters 1-7 of Walter Rudin's "Principles of Mathematical Analysis"](https://math.berkeley.edu/~gbergman/ug.hndts/#Rudin) - George M. Bergman
+* [Calculus Made Easy](http://calculusmadeeasy.org/) - Silvanus P. Thompson
```
Comment (rossant) [inline README.md:384]: idem (1910)

File: README.md [hunk 12]
```diff
@@ -365,6 +381,8 @@ A curated list of awesome mathematics resources.
...
+* [Elemens of Differential and Integral Calculus](http://djm.cc/library/Elements_Differential_Integral_Calculus_Granville_edited_2.pdf) - William Anthony Granville (PDF)
```
Comment (rossant) [inline README.md:385]: idem (1911)
also typo: `ElemenTs`

File: README.md [hunk 13]
```diff
@@ -420,6 +439,7 @@ A curated list of awesome mathematics resources.
 * [Elementary Differential Equations](http://digitalcommons.trinity.edu/mono/8/) - William F. Trench
 * [Elementary Differential Equations With Boundary Value Problems](http://digitalcommons.trinity.edu/mono/9/) - William F. Trench
 * [Notes on Diffy Qs: Differential Equations for Engineers](http://www.jirka.org/diffyqs/) - Jiří Lebl
+* [Differential Equations](http://djm.cc/library/Differential_Equations_Phillips_edited.pdf) - H. B. Phillips (PDF)
```
Comment (rossant) [inline README.md:442]: idem (1922)

File: README.md (hunk 1 above)
Comment (jackdolgin) [reply README.md:139]: sure, happy to! can do so later today

File: README.md (hunk 12 above)
Comment (jackdolgin) [reply README.md:385]: thanks for the catch

**Comment by rossant** [comment]
Thanks for the many additions!

**Pull request merged**